packages/circuits/src/merkle.circom

```diff
@@ -191,4 +191,27 @@ template ConstructRoot(levels) {
     }
 
     root <== hashers[levels - 1].hash;
+}
+
+template CheckMerkleRoot(levels) {
+    signal input root;
+    signal input leaves[2**levels];
```

Comment:
Big nitpick, but when I imagine a "two-level Merkle tree", I imagine one with 3 nodes (first level = root, second level = leaves). This seems to take `levels` as being the non-root levels?

Reply:
That's what I thought initially too, but seems like the other circuits use `levels` as non-root levels. Thinking a bit deeper, I think it might make sense so we can define all the adjacent hash path arrays as size `levels` rather than size `levels - 1`. What do you think?

Reply:
Makes sense :)